wallet/wallet.go

```diff
@@ -2848,56 +2823,12 @@ type GetTransactionsResult struct {
 // therefore the notes and restrictions of that function also apply here.
 func (w *Wallet) GetTransactions(ctx context.Context, f func(*Block) (bool, error), startBlock, endBlock *BlockIdentifier) error {
 	const op errors.Op = "wallet.GetTransactions"
-	var start, end int32 = 0, -1
-
-	if startBlock != nil {
-		if startBlock.hash == nil {
-			start = startBlock.height
-		} else {
-			err := walletdb.View(ctx, w.db, func(dbtx walletdb.ReadTx) error {
-				ns := dbtx.ReadBucket(wtxmgrNamespaceKey)
-				serHeader, err := w.TxStore.GetSerializedBlockHeader(ns, endBlock.hash)
```

Comment:
^^^
Bug alluded to is this. This should have been `startBlock.hash`. Never affected Decrediton since we always range over heights (vs hashes) but could eventually bite someone.